[Bug] Remove the start slash of the assembly descriptor for the presto and es plugins

### Search before asking

- [X] I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar issues.


### Linkis Component

linkis-engineconnn-plugin

### Steps to reproduce

Executing `mvn clean install -Dmaven.javadoc.skip=true -DskipTests` displays the following error message, but does not affect packaging.
```
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /dist/v${presto.version}/conf
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /plugin/${presto.version}
[INFO] Copying files to F:\tmp\linkis\incubator-linkis\linkis-engineconn-plugins\presto\target\out
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /dist/v${presto.version}/conf
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /plugin/${presto.version}

[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /dist/v${elasticsearch.version}/conf
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /plugin/${elasticsearch.version}
[INFO] Copying files to F:\tmp\linkis\incubator-linkis\linkis-engineconn-plugins\elasticsearch\target\out
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /dist/v${elasticsearch.version}/conf
[ERROR] OS=Windows and the assembly descriptor contains a *nix-specific root-relative reference (starting with slash): /plugin/${elasticsearch.version}
```

### Expected behavior

No

### Your environment

<!-- Please describe the linkis version you are using and basic environment information -->
<!-- 请描述您使用的linkis版本和基本环境信息 -->
- Linkis version used: 1.2.0

### Anything else

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

--- a/linkis-engineconn-plugins/elasticsearch/src/main/assembly/distribution.xml
+++ b/linkis-engineconn-plugins/elasticsearch/src/main/assembly/distribution.xml
@@ -51,7 +51,7 @@
                 <include>log4j2.xml</include>
             </includes>
             <fileMode>0777</fileMode>
-            <outputDirectory>/dist/v${elasticsearch.version}/conf</outputDirectory>
+            <outputDirectory>dist/v${elasticsearch.version}/conf</outputDirectory>
             <lineEnding>unix</lineEnding>
         </fileSet>
 
@@ -64,7 +64,7 @@
                 <exclude>*doc.jar</exclude>
             </excludes>
             <fileMode>0777</fileMode>
-            <outputDirectory>/plugin/${elasticsearch.version}</outputDirectory>
+            <outputDirectory>plugin/${elasticsearch.version}</outputDirectory>
         </fileSet>
 
     </fileSets>

--- a/linkis-engineconn-plugins/presto/src/main/assembly/distribution.xml
+++ b/linkis-engineconn-plugins/presto/src/main/assembly/distribution.xml
@@ -51,7 +51,7 @@
                 <include>log4j2.xml</include>
             </includes>
             <fileMode>0777</fileMode>
-            <outputDirectory>/dist/v${presto.version}/conf</outputDirectory>
+            <outputDirectory>dist/v${presto.version}/conf</outputDirectory>
             <lineEnding>unix</lineEnding>
         </fileSet>
 
@@ -64,7 +64,7 @@
                 <exclude>*doc.jar</exclude>
             </excludes>
             <fileMode>0777</fileMode>
-            <outputDirectory>/plugin/${presto.version}</outputDirectory>
+            <outputDirectory>plugin/${presto.version}</outputDirectory>
         </fileSet>
 
     </fileSets>

